Mapfile Layergroupnames in upgrade 2.2 -> 2.2.1

During upgrade to GMF 2.2.1 it get following Error:
```
Traceback (most recent call last):
  File ".build/venv/bin/pot-create", line 11, in <module>
    sys.exit(main())
  File "/home/svagap/mapbs5/.build/venv/local/lib/python2.7/site-packages/lingua/extract.py", line 349, in main
    entry.update(message, add_occurrences=options.location)
  File "/home/svagap/mapbs5/.build/venv/local/lib/python2.7/site-packages/lingua/extract.py", line 73, in update
    self.occurrences.append((message.location[0], str(message.location[1])))
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe4' in position 7: ordinal not in range(128)
CONST_Makefile:725: recipe for target 'mapbs/locale/mapbs-client.pot' failed
make[1]: *** [mapbs/locale/mapbs-client.pot] Error 1
make[1]: Leaving directory '/home/svagap/mapbs5'

================================================================
The command '['make', '-f', 'svagap221.mk', 'build']' returns the error code 2.
Fix it and run it again:
make -f svagap221.mk upgrade4
```

I can remove the error if I rename all wms_layer_group (remove ä/ö/ü etc) 
```
 # --------------------------------------------------------------------------- 
  LAYER
    NAME "NG_Intensitaetskarten_ProzessWasserHQ30"
    TYPE POLYGON
	EXTENT ${layer_extent}
    CONNECTIONTYPE postgis
	PROCESSING "CLOSE_CONNECTION=DEFER"
	CONNECTION "${mapserver_connection}"
    DATA "the_geom FROM (SELECT hist_id as the_id, intensitaetsklasse as ng_intensitaetsklasse, the_geom
                          FROM ng_naturgefahren.hv_ng_in_intensitaeten
						  WHERE prozess like 'Ueberschwemmung' AND jaehrlichkeit like 'HQ30') 
                         AS str1 USING UNIQUE the_id USING SRID=2056"
	
	PROJECTION
		"init=epsg:2056"
	END

	METADATA
        "ows_extent" "${layer_extent}"
        "ows_title" "Prozess Wasser HQ30"
        "wms_layer_group" "/Naturgefahren/Intensitätskarten" **<-- replace ä by ae**
        "wms_srs" "EPSG:2056 EPSG:21781 EPSG:4326 EPSG:3857 EPSG:4258"
        "gml_include_items" "all"
		"ows_the_geom_type" "multipolygon"
		"ows_geometries" "the_geom"
        "gml_featureid" "the_id"
    END
	
	COMPOSITE
		OPACITY 70
    END
		
	CLASSITEM "ng_intensitaetsklasse"
```
Maybe the file should not be interpreted as ascii but utf-8. We do not want to rename all layer_group just to get a correct translation.

--- a/c2cgeoportal/lib/lingua_extractor.py
+++ b/c2cgeoportal/lib/lingua_extractor.py
@@ -180,6 +180,8 @@ class GeoMapfishConfigExtractor(Extractor):  # pragma: no cover
     extensions = [".yaml"]
 
     def __call__(self, filename, options):
+        init_region({"backend": "dogpile.cache.memory"})
+
         with open(filename) as config_file:
             from c2cgeoportal.lib.print_ import *  # noqa
             config = yaml.load(config_file)
@@ -406,11 +408,13 @@ class GeoMapfishThemeExtractor(Extractor):  # pragma: no cover
         attributes, layers = self._layer_attributes(url, layer)
         for layer in layers:
             messages.append(Message(
-                None, layer, None, [], "", "", (".".join([item_type, name]), layer)
+                None, layer, None, [], "", "",
+                (".".join([item_type, name]), layer.encode("ascii", "replace"))
             ))
         for attribute in attributes:
             messages.append(Message(
-                None, attribute, None, [], "", "", (".".join([item_type, name]), layer)
+                None, attribute, None, [], "", "",
+                (".".join([item_type, name]), layer.encode("ascii", "replace"))
             ))
 
     def _layer_attributes(self, url, layer):

